pyanaconda/ui/gui/spokes/welcome.py

```diff
@@ -162,6 +162,10 @@ def initialize(self):
         # get language codes for the locales
         langs = [localization.get_language_id(locale) for locale in locales]
 
+        # get common languages and append them here
+        common_locales = localization.get_common_locales()
+        langs += [localization.get_language_id(locale) for locale in common_locales]
```

Comment:
Will this duplicate languages on territories where a common language is a territory languace too?

Reply:
no.. the language will appear once.. in territory and be omitted from common.